### PR TITLE
Implement @SqlScript

### LINKED ISSUE #131: Multiple SqlBatch annotations
Hi, it would be convenient if we could assign more than one SqlBatch annotations to a method of a DAO class.

Example:

``` java
    @SqlBatch("CREATE TABLE IF NOT EXISTS <table> (`id` int(11), `name` varchar(16))")
    @SqlBatch("ALTER TABLE <table> ADD COLUMN `id` int(11)")
    @SqlBatch("ALTER TABLE <table> ADD COLUMN `name` varchar(16)")
    void createTable(@Define("table") tableName);
```

### PR DESCRIPTION
**Pull request by stevenschlansker**
This allows you to execute multiple statements in a batch

Fixes #131

### CODE REVIEW

**Comment by arteam** [comment]
`@SqlScript` with actual SQL doesn't seem very useful. Usually SQL scripts tend to be quite big, sometimes they're generated by a database dump. I think we should provide a way to specify a path to an SQL script from the classpath.

**Comment by stevenschlansker** [comment]
I tend to agree with you, but it was an explicit feature request.  I can see some use cases where 2-3 lines are nicer inline than externalized.  But that said, it should work just as well with a sql locator... I'll add a test showing that!

**Comment by stevenschlansker** [comment]
@arteam now it supports loading from classpath via `@UseClasspathSqlLocator`, is that more compelling?

File: sqlobject/src/test/resources/org/jdbi/v3/sqlobject/statement/TestSqlScripts$Scriptacular/externalScript.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CREATE TABLE temp (pk integer primary key);
```
Comment (arteam) [inline sqlobject/src/test/resources/org/jdbi/v3/sqlobject/statement/TestSqlScripts$Scriptacular/externalScript.sql]: I guess the license checker wants too see a license boilerplate in the file header.

**Comment by arteam** [comment]
Looks great now!

**Pull request merged**